src/qibo/imaging/mpldrawer.py

```diff
@@ -0,0 +1,481 @@
+# Some functions in MPLDrawer are from code provided by Rick Muller
+# Simplified Plotting Routines for Quantum Circuits
+# https://github.com/rpmuller/PlotQCircuit
+#
+import matplotlib
+import numpy as np
+
+
+class MPLDrawer:
+    def __init__(self):
+        pass
```

Comment:
This class has no explicit attributes, and they are not even set later on (which is a good thing).

Just remove these three lines, deindent everything else, and remove `self` from functions parameters and methods calls.
It will work in the exact same way, saving a bunch of overhead.

Reply:
Thanks! I have removed the class and now there is a `ui` submodule with cleaner code

Reply:
Done in ffdf9b0b48439d174623131be8aaf5d3392ac72f

Reply:
Whenever you complete something, feel free to add a comment like the former, and resolve the conversation.